code/modules/mob/living/simple_animal/hostile/terror_spiders/empress.dm

```diff
@@ -0,0 +1,241 @@
+	else
+		to_chat(src, "You reach through bluespace, searching for organic minds... but find none nearby.")
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EmpressToggleDebug()
+	set name = "Toggle Debug"
+	set category = "Spider"
+	set desc = "Enables/disables debug mode for spiders."
+	if (spider_debug)
+		spider_debug = 0
+		to_chat(src, "Debug: DEBUG MODE is now <b>OFF</b> for all spiders in world.")
+	else
+		spider_debug = 1
+		to_chat(src, "Debug: DEBUG MODE is now <b>ON</b> for all spiders in world.")
+	for(var/mob/living/simple_animal/hostile/poison/terror_spider/T in mob_list)
+		T.spider_debug = spider_debug
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EmpressToggleInstant()
+	set name = "Toggle Instant"
+	set category = "Spider"
+	set desc = "Enables/disables instant growth for spiders."
+	if (spider_growinstantly)
+		spider_growinstantly = 0
+		to_chat(src, "Debug: INSTANT GROWTH is now <b>OFF</b> for all spiders in world.")
+	else
+		spider_growinstantly = 1
+		to_chat(src, "Debug: INSTANT GROWTH is now <b>ON</b> for all spiders in world.")
+	for(var/mob/living/simple_animal/hostile/poison/terror_spider/T in mob_list)
+		T.spider_growinstantly = spider_growinstantly
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EmpressKillSpider()
+	set name = "Erase Spider"
+	set category = "Spider"
+	set desc = "Kills a spider. If they are player-controlled, also bans them from controlling any other spider for the rest of the round."
+	var/choices = list()
+	for(var/mob/living/simple_animal/hostile/poison/terror_spider/L in mob_list)
+		if(L == src)
+			continue
+		if (L.health < 1)
+			continue
+		choices += L
+	var/killtarget = input(src,"Which terror spider should die?") in null|choices
+	if (!killtarget)
+		// cancel
+	else if (!isliving(killtarget))
+		to_chat(src, "[killtarget] is not living.")
+	else if (!istype(killtarget, /mob/living/simple_animal/hostile/poison/terror_spider/))
+		to_chat(src, "[killtarget] is not a terror spider.")
+	else
+		var/mob/living/simple_animal/hostile/poison/terror_spider/T = killtarget
+		if (T.ckey)
+			// living player
+			ts_ckey_blacklist += T.ckey
+		to_chat(T, "<span class='userdanger'> Through the hivemind, the raw power of [src] floods into your body, burning it from the inside out! </span>")
+		T.death()
+		T.gib()
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EraseBrood()
+	set name = "Erase Brood"
+	set category = "Spider"
+	set desc = "Debug: kill off all other spiders in the world. Takes two minutes to work."
+	for(var/mob/living/simple_animal/hostile/poison/terror_spider/T in mob_list)
+		if (T.spider_tier < 5)
+			T.degenerate = 1
+			T.loot = 0
+			to_chat(T, "<span class='userdanger'> Through the hivemind, the raw power of [src] floods into your body, burning it from the inside out! </span>")
+	for(var/obj/effect/spider/terror_eggcluster/T in world)
```

Comment:
NONONONONONO. Never use in wolrd loops if you can help it. Just make a list for these things, and add them to it in New(), while removing them in Destroy().